media-plugins/gst-plugins-qt5/gst-plugins-qt5-1.18.4.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+GST_ORG_MODULE=gst-plugins-good
+
+inherit gstreamer-meson
+
+DESCRIPTION="Qt5 QML video sink plugin"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="+egl wayland +X orc"
+
+REQUIRED_USE="
+	wayland? ( egl )
+"
+
+GST_REQ="${PV}"
+RDEPEND="
+	>=media-libs/gst-plugins-base-${GST_REQ}:${SLOT}[egl?,opengl,wayland?,X?]
+	media-libs/mesa[egl?,X?,${MULTILIB_USEDEP}]
+
+	dev-qt/qtcore[${MULTILIB_USEDEP}]
+	dev-qt/qtgui[${MULTILIB_USEDEP}]
+	egl? ( dev-qt/qtgui[eglfs] )
+	dev-qt/qtdeclarative[${MULTILIB_USEDEP}]
+	wayland? ( dev-qt/qtwayland[${MULTILIB_USEDEP}] )
+	X? ( dev-qt/qtx11extras[${MULTILIB_USEDEP}] )
+"
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/gst-dont-use-volatile-to-mean-atomic-1.18.4.patch"
+)
+
+GST_PLUGINS_ENABLED="qt5"
```

Comment:
This is the default value for split ebuilds (ones where `PN != GST_ORG_MODULE`).

Reply:
Removing this line seems to not work, did you mean to use the literal `${PN/gst-plugins-/}`?